src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Entities/File.cs

```diff
@@ -22,4 +22,8 @@ public File()
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     [JsonPropertyName("mediaType")]
     public object MediaType { get; set; }
+
+    [JsonRequired]
+    [JsonPropertyName("name")]
+    public override string Name { get; set; }
```

Comment:
What is the motivation behind overriding here instead of just inheriting this property from the parent class? Do we just want to have different JSON tags in this class? Same question for `VerifiedUsing` in `NTIAFile`